src/viam/rpc/dial.py

```diff
@@ -234,7 +241,7 @@ def release():
         channel = ViamChannel(chan, release)
         return channel
 
-    runtime.release()
+    await runtime.release()
```

Comment:
`runtime.release()` is not an awaitable method

Reply:
Whoops, this is left over from testing async PointerCounter. Sorry